package.json

```diff
@@ -1,37 +1,37 @@
 {
-  "name": "gatsby-example-using-markdown-pages",
+  "name": "cycle-7-website",
   "private": true,
   "description": "Start your new blog using markdown files",
   "version": "0.1.0",
   "author": "@gatsbyjs",
   "dependencies": {
-    "@buildit/gravity-ui-web": "next",
-    "@mdx-js/mdx": "^1.5.3",
-    "@mdx-js/react": "^1.5.3",
-    "gatsby": "^2.18.10",
-    "gatsby-image": "^2.2.18",
-    "gatsby-plugin-manifest": "^2.2.16",
-    "gatsby-plugin-mdx": "^1.0.64",
-    "gatsby-plugin-offline": "^2.2.10",
-    "gatsby-plugin-react-helmet": "^3.1.7",
-    "gatsby-plugin-sass": "^2.1.14",
-    "gatsby-plugin-sharp": "^2.2.21",
-    "gatsby-remark-autolink-headers": "^2.1.21",
-    "gatsby-remark-custom-blocks": "^2.1.21",
-    "gatsby-source-filesystem": "^2.1.22",
-    "gatsby-transformer-remark": "^2.6.42",
-    "gatsby-transformer-sharp": "^2.2.14",
-    "node-sass": "^4.12.0",
+    "@buildit/gravity-ui-web": "^3.0.0",
+    "@mdx-js/mdx": "^1.5.7",
+    "@mdx-js/react": "^1.5.7",
+    "gatsby": "^2.20.7",
+    "gatsby-image": "^2.3.1",
+    "gatsby-plugin-manifest": "^2.3.3",
+    "gatsby-plugin-mdx": "^1.1.3",
+    "gatsby-plugin-offline": "^3.1.2",
+    "gatsby-plugin-react-helmet": "^3.2.1",
+    "gatsby-plugin-sass": "^2.2.1",
+    "gatsby-plugin-sharp": "^2.5.3",
+    "gatsby-remark-autolink-headers": "^2.2.1",
+    "gatsby-remark-custom-blocks": "^2.2.1",
+    "gatsby-source-filesystem": "^2.2.2",
+    "gatsby-transformer-remark": "^2.7.1",
+    "gatsby-transformer-sharp": "^2.4.2",
+    "node-sass": "^4.13.1",
     "prop-types": "^15.7.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
     "react-helmet": "^5.2.1",
-    "react-scroll": "^1.7.14"
+    "react-scroll": "^1.7.16"
   },
   "devDependencies": {
-    "eyeglass": "^2.4.1",
-    "gatsby-plugin-google-tagmanager": "^2.1.12",
-    "prettier": "^1.17.0"
+    "eyeglass": "^2.5.0",
```

Comment:
I think this can be removed altogether now?

Reply:
Indeed it can. On it.

Reply:
done